app/src/main/res/layout/fragment_transaction_details_action_multisend.xml

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.coordinatorlayout.widget.CoordinatorLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:background="@color/white">
+
+    <com.google.android.material.appbar.AppBarLayout
```

Comment:
It seems we have this same snippet in several place only with different title texts. Can we replace this with a custom view? Or an include if it is possible to parametrise them. This change is probably outside the scope of this PR.

Reply:
we can think about it at some later point. With view binding, we do not have a possibility to pass arguments to the included layout. I am against enabling data binding as it will increase compile time. A custom view could be an option. Right now I don't mind this small duplication as it makes layouts more explicit and flexible.